### PR TITLE
Make non-global constants have consistent naming (with just _ instead of _k)

### PR DESCRIPTION
**Pull request by gspencergoog**
Our style guide says the k's are not necessary, and it seems like a good idea to make all the code be consistent on this.

Only naming changes to private vars: no logic changes.

### CODE REVIEW

**Comment by gspencergoog** [comment]
cc @Hixie for review

File: packages/flutter/lib/src/services/message_codecs.dart [hunk 1]
```diff
@@ -245,20 +245,20 @@ class StandardMessageCodec implements MessageCodec<dynamic> {
...
+  static const int _int64 = 4;
+  static const int _largeInt = 5;
+  static const int _float64 = 6;
+  static const int _string = 7;
+  static const int _uint8List = 8;
+  static const int _int32List = 9;
+  static const int _int64List = 10;
+  static const int _float64List = 11;
+  static const int _list = 12;
+  static const int _map = 13;
```
Comment (Hixie) [inline packages/flutter/lib/src/services/message_codecs.dart]: for things like this I think we should have a common prefix or suffix to indicate that they are related.

**Comment by xqwzts** [comment]
While the [style guide wiki](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k) says "It's not necessary to add the k prefix to non-global constants.", it also contains an example of a private var beginning with `_k`:

> const Color _kBarrierColor = Colors.black54;

Perhaps we could remove that to avoid confusion?

File: packages/flutter_driver/lib/src/driver/driver.dart [hunk 2]
```diff
@@ -128,10 +128,10 @@ class FlutterDriver {
...
        _driverId = _nextDriverId++;
 
-  static const String _kFlutterExtensionMethod = 'ext.flutter.driver';
-  static const String _kSetVMTimelineFlagsMethod = '_setVMTimelineFlags';
-  static const String _kGetVMTimelineMethod = '_getVMTimeline';
-  static const String _kClearVMTimelineMethod = '_clearVMTimeline';
+  static const String _flutterExtensionMethod = 'ext.flutter.driver';
+  static const String _setVMTimelineFlagsMethod = '_setVMTimelineFlags';
+  static const String _getVMTimelineMethod = '_getVMTimeline';
+  static const String _clearVMTimelineMethod = '_clearVMTimeline';
```
Comment (Hixie) [inline packages/flutter_driver/lib/src/driver/driver.dart]: maybe s/Method/MethodName/

**Comment by gspencergoog** [comment]
At the top level scope, the underscore prefix doesn't mean that it's private, it's just a global that is [limited to the library's scope](https://www.dartlang.org/guides/language/language-tour#libraries-and-visibility).

**Comment by Hixie** [comment]
LGTM

Not sure how much of this is a win, sometimes it's nice to be able to easily tell if the identifier is some private field (with a value that varies from call to call), or some constant (where it's unambiguous that the value is always the same every time).

**Comment by gspencergoog** [comment]
Well, but didn't we change the style guide to relax the _k requirement for a reason? Was it to be consistent with the Dart style guide?

Also, I think the style guide should be more definitive: we should either say "don't use the _k prefix for anything other than globals" or "always use _k for all constants". Having it be optional for some usages adds no value, which is where I was headed with this PR: normalizing the usage so we can tighten up that rule to just be "only use _k for globals".  I don't really care which way we go, but it should be consistent.

File: packages/flutter/lib/src/services/message_codecs.dart (hunk 1 above)
Comment (gspencergoog) [reply packages/flutter/lib/src/services/message_codecs.dart]: Done. (named them all _valueFoo)

File: packages/flutter_driver/lib/src/driver/driver.dart (hunk 2 above)
Comment (gspencergoog) [reply packages/flutter_driver/lib/src/driver/driver.dart]: Done.

**Pull request merged**